Source/Scene/ModelExperimental/ModelExperimentalNode.js

```diff
@@ -22,6 +26,26 @@ export default function ModelExperimentalNode(options) {
   Check.typeOf.object("options.modelMatrix", options.modelMatrix);
   //>>includeEnd('debug');
 
+  /**
+   * The scene graph this node belongs to.
+   *
+   * @type {ModelExperimentalSceneGraph}
+   * @readonly
+   *
+   * @private
+   */
+  this.sceneGraph = options.sceneGraph;
+
+  /**
+   * The indices of the children of this node in the runtime nodes array of the scene graph.
```

Comment:
Will users expect the exact same indices as in the glTF? Because the `runtimeNodes` array is generated from a post-order depth-first traversal of the scene graph, a node's glTF ID will not necessarily be a match to its index in the `runtimeNodes` array.